task.go

```diff
@@ -55,6 +69,9 @@ type task struct {
 	io          *IO
 	containerID string
 	pid         uint32
+
+	deferedRequest *execution.CreateRequest
```

Comment:
`defered` -> `deferred`, or wait for someone to contribute this change tomorrow :)